### PR TITLE
Appium find by image support

### PR DESCRIPTION
**Pull request by jayandran-Sampath**
## Proposed changes

Adding support for appium image selector 

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Comment by jayandran-Sampath** [comment]
@christian-bromann  Hey, I closed the Previous PR due to rebase issue with the branch. 

I have changed the approach by defining stragegy based on string which ends with. 

Kindly look into it

File: packages/webdriverio/src/utils/findStrategy.js [hunk 1]
```diff
@@ -19,6 +19,9 @@ const XPATH_SELECTOR_REGEXP = [
     // *=query or =query
     /(\*)?=(.+)$/,
 ]
+const IMAGEPATH_MOBILE_SELECTORS_ENDSWITH = [
+    '.jpg', '.gif', '.png', '.bmp', '.svg'
```
Comment (christian-bromann) [inline packages/webdriverio/src/utils/findStrategy.js:23]:
```suggestion
    '.jpg', '.jpeg', '.gif', '.png', '.bmp', '.svg'
```

File: docs/Selectors.md [hunk 2]
```diff
@@ -341,6 +341,19 @@ With selector chaining, it's way easier. Simply narrow down the desired element
...
+### Appium Image Selector
+
+Using the  `-image` locator strategy, it is possible to send an Appium an image file representing an element you want to access.
+
+Supported file formats `jpg,png,gif,bmp,svg`
+
+Full reference can be found [here](https://github.com/appium/appium/blob/master/docs/en/advanced-concepts/image-elements.md)
+
+```js
+const elem = $('filepathofimage/test.jpg')
```
Comment (christian-bromann) [inline docs/Selectors.md:353]:
```suggestion
const elem = $('./file/path/of/image/test.jpg')
```

**Comment by christian-bromann** [comment]
The build is broken due to an issue that was fixed in master. I will go ahead and merge.

Thanks for adding this feature @jayandran-Sampath and congrats on your first WebdriverIO contribution! 🎉

**Comment by khoingo-lg** [comment]
My locator is {accessibility: '~abc.png'}.
It works well with the old strategy (webdriverio 5.23.0). After I upgrade the new version webdriverio 6.3.4, webdriverIO use image strategy for my locator. It does not work. It throws element not found.

My environment: 
Appium: 1.17.1
Webdriverio: 6.3.4
Device: iOS simulator 13.3

Could you have any advice for me?

**Pull request merged**